src/Render.cc

```diff
@@ -16,11 +18,15 @@ using namespace snowcrash;
 namespace drafter {
 
     // JSON content-type regex
-    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json";
+    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json[[:blank:]]*(;.*|$)";
```

Comment:
There's a test `application/schema+json ; hello=world`. Not exactly for `application/json` but it does test the added regex part.